lib/ngx/errlog.lua

```diff
@@ -87,4 +89,10 @@ function _M.get_logs(max, logs)
 end
 
 
+function _M.get_sys_filter_level()
+    local r = getfenv(0).__ngx_req
+    return tonumber(C.ngx_http_lua_ffi_errlog_get_sys_filter_level(r))
```

Comment:
This function may on the hot Lua code paths and we should avoid use of the Lua global variables `getfenv` and `tonumber` in this Lua function body. Better cache their values local variables in an upper-level scope.